omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -348,6 +350,10 @@ Client Details:
     # the formerowner from group, just unsets the ownership. This works fine
     # with Create and Edit Group privilege only
     $ bin/omero user leavegroup testgroup --name formerowner  --as-owner
+    # Move a Dataset to a group where the prospective owner is member
+    # and include all annotations on objects
+    # linked to that Dataset even if they do not belong to the same user
+    $ bin/omero chgrp 5 Dataset:51 --include Annotation
     # Transfer ownership to user 55 of the whole hierarchy of the Project 112
     # including the links, as long as all the objects and links in the
     # hierarchy belong to the same user
```

Comment:
@mtbc : Expected behaviour here is that in RA and RW groups, the annotations on the hierarchy will not be transferred, but stay with the original user. This does not depend on whether or not the owner of the data is in the group, does it ?

Second question: Chgrp of a Dataset as described above https://github.com/openmicroscopy/ome-documentation/pull/1617/commits/753f70c60809b6b6e78cf7d564a3827e97aec719#diff-dc981c0425062d2436a7cf30eff329f7R353 is going to behave as described if all the Datasets and Images and links belong to the same user (annotations do not have to, I chacked, the --include will override all). But what if an image (with annotations) in that Dataset belongs to somebody else ? Will the Chgrp procced as described ?